<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>451</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>451</width>
    <height>680</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AFD Word Checker 1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="fondo_blanco">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>481</width>
      <height>741</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F7F7E6;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupTransiciones">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Transiciones</string>
    </property>
    <widget class="QTableWidget" name="tabla_transiciones">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>256</width>
       <height>85</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>74</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
    <widget class="QLineEdit" name="in_origen">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_agregar_transicion">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>32</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Para agregar una transición, debe &lt;span style=&quot; text-decoration: underline;&quot;&gt;indicar&lt;/span&gt; lo siguiente:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Origen&lt;/span&gt;:   Estado del comienzo de la transición.&lt;br&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Símbolo&lt;/span&gt;: Símbolo de la transición.&lt;br&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Destino&lt;/span&gt;: Estado donde finaliza la transición.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Agregar nueva</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_destino">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_simbolo">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Origen  Símbolo  Destino</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_eliminar_select">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>75</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar
transicion
seleccionada</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bton_eliminar_todas">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar todas</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_estados">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Estados</string>
    </property>
    <widget class="QLineEdit" name="inEstadoInicial">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>110</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inEstadoFinal">
     <property name="geometry">
      <rect>
       <x>234</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>37</x>
       <y>34</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Estado inicial</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botonSetEstadoInicial">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>140</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click aquí para establecer un estado inicial (debe escribir el nombre del estado en el cuadro de arriba). Reemplazará al estado inicial previo. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Establecer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botonAgregarFinal">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Agregar a la lista</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>35</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> Estados finales</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botonQuitarEstadoFinal">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>105</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Quitar
estado final
seleccionado</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEstadoInicial">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>60</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aquí se mostrará el estado inicial que se ha establecido. En caso de que no se haya establecido uno, se mostrará un &amp;quot;?&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:solid;
border-color:black;
border-width:2px;</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="listaEstadosFinales">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupPalabra">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Palabra</string>
    </property>
    <widget class="QLineEdit" name="inPalabra">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="botonProbarPalabra">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click para comenzar a probar la palabra.&lt;br/&gt;El resultado se mostrará en la salida.&lt;/p&gt;&lt;p&gt;Detalles de la prueba en &amp;quot;salida detallada&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Probar!</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Salida Simple</string>
      </attribute>
      <widget class="QLabel" name="salida2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>351</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Salida detallada</string>
      </attribute>
      <widget class="QTextEdit" name="salida">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>391</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:black;
color:#00ff00;</string>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inEstadoInicial</tabstop>
  <tabstop>botonSetEstadoInicial</tabstop>
  <tabstop>inEstadoFinal</tabstop>
  <tabstop>botonAgregarFinal</tabstop>
  <tabstop>listaEstadosFinales</tabstop>
  <tabstop>botonQuitarEstadoFinal</tabstop>
  <tabstop>in_origen</tabstop>
  <tabstop>in_simbolo</tabstop>
  <tabstop>in_destino</tabstop>
  <tabstop>boton_agregar_transicion</tabstop>
  <tabstop>tabla_transiciones</tabstop>
  <tabstop>boton_eliminar_select</tabstop>
  <tabstop>inPalabra</tabstop>
  <tabstop>botonProbarPalabra</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>bton_eliminar_todas</tabstop>
  <tabstop>salida</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
